<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-F1GDK7B" failures="1" tests="9" name="Test" time="0.19" errors="0" timestamp="2023-09-01T10:51:20 IST">
  <testcase classname="demo.NewTest" name="setData" time="0.011"/>
  <testcase classname="demo.NewTest" name="setData" time="0.009"/>
  <testcase classname="demo.NewTest" name="setData" time="0.002"/>
  <testcase classname="demo.NewTest" name="testcase1" time="0.002"/>
  <testcase classname="demo.NewTest" name="testcase5" time="0.002">
    <skipped/>
  </testcase> <!-- testcase5 -->
  <testcase classname="demo.NewTest" name="testcase3" time="0.02">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.NewTest.testcase3(NewTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase3 -->
  <testcase classname="demo.NewTest" name="testcase7" time="0.001">
    <skipped/>
  </testcase> <!-- testcase7 -->
  <testcase classname="demo.NewTest" name="testcase2" time="0.002"/>
  <testcase classname="demo.NewTest" name="testcase6" time="0.005"/>
  <testcase classname="demo.NewTest" name="testcase4" time="0.0">
    <ignored/>
  </testcase> <!-- testcase4 -->
</testsuite> <!-- Test -->
